### PR TITLE
Fix eslint 'no-useless-escape' errors. (#7747)

### PR DESCRIPTION
**Pull request by Shivansh2407**
## Description
Fix eslint 'no-useless-escape' errors. #7747

### CODE REVIEW

**Comment by Shivansh2407** [comment]
But these are a part of regex , so please let me know the method to fix them.

**Comment by amychurchwell** [comment]
@Shivansh2407 
Screen Shot 2019-09-19 at 8 37 45 AM
If you navigate to the files and line numbers listed in the `make lint` errors- you should see underlines underneath the problem characters in your code editor like above (I'm using [Visual Studio Code](https://code.visualstudio.com/)). If you hover over these it will confirm that it is an unnecessary character to be deleted.

**Comment by amychurchwell** [comment]
Hi @Shivansh2407! we don't need to open a separate PR for every file, they can be added to your original PR.

**Comment by amychurchwell** [comment]
Please commit and push the other file changes to this original branch `patch-3` and this PR will be updated. Thanks again, Shivansh!

**Comment by amychurchwell** [comment]
@Shivansh2407 are you still working on this pull request?

**Comment by Shivansh2407** [comment]
Yes I am working on it

**Comment by Shivansh2407** [comment]
I have completed the requested changes.Please check and let me know the details.

File: media/js/firefox/whatsnew/experiment-whatsnew-68-fr.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     // courtesy of https://davidwalsh.name/query-string-javascript
     function getUrlParam(name) {
-        name = name.replace(/[\[]/, '\\[').replace(/[\]]/, '\\]');
```
Comment (amychurchwell) [inline media/js/firefox/whatsnew/experiment-whatsnew-68-fr.js]: Not every slash here is a `no-useless-escape` error.

If you look in your text editor you should see something similar to:
Screen Shot 2019-10-07 at 8 11 59 AM
Note how one of the slashes has a double yellow underline?
Screen Shot 2019-10-07 at 8 15 05 AM
That is the only one that needs to be removed. Removing others breaks the regex and is creating test failures.

Comment (amychurchwell) [reply media/js/firefox/whatsnew/experiment-whatsnew-68-fr.js]: This feedback also applies to:
- `experiment-whatsnew-68-enUS.js`
- `experiment-whatsnew-68-enGB.js`
- `experiment-whatsnew-68-enCA.js`
- `experiment-whatsnew-68-de.js`

Comment (Shivansh2407) [reply media/js/firefox/whatsnew/experiment-whatsnew-68-fr.js]: Ok Cool I will fix this right away..

**Comment by amychurchwell** [comment]
Hi @Shivansh2407, you don't need to close the pull request. You can push a commit to the same branch/pull request with any updates needed.

**Comment by Shivansh2407** [comment]
Please Check now #7838 @amychurchwell .... Thanks

**Pull request closed without merging**